github/projects.go

```diff
@@ -79,28 +79,25 @@ type ListProjectsOptions struct {
 //
 // GitHub API docs: https://docs.github.com/rest/projects/fields
 type ProjectV2FieldOption struct {
-	ID string `json:"id,omitempty"`
-	// The display name of the option.
-	Name string `json:"name,omitempty"`
-	// The color associated with this option (e.g., "blue", "red").
-	Color string `json:"color,omitempty"`
-	// An optional description for this option.
-	Description string `json:"description,omitempty"`
+	ID          *int64 `json:"id,omitempty"`          // The unique identifier for this option.
+	Name        string `json:"name,omitempty"`        // The display name of the option.
+	Color       string `json:"color,omitempty"`       // The color associated with this option (e.g., "blue", "red").
+	Description string `json:"description,omitempty"` // An optional description for this option.
```

Comment:
Optional fields should be pointers to strings in general. I'm not sure why these were left as values in #3718.